projects/uitdatabank/docs/terminology.md

```diff
@@ -2,8 +2,82 @@
 
 On this page you can get a grasp of common UiTdatabank terminology and concepts, referenced throughout the API documentation.
 
-<!-- theme: warning -->
+## Event
 
-> ##### Construction ahead 🚧
->
-> This page is a placeholder and under construction.
+An API resource with a unique id that represents a cultural, leisure or sport-related activity. Usually happening on one or multiple specific dates and times, or on a pre-defined weekly schedule.
+
+Most events will be published in [online calendars](#online-calendar) so potential attendees can find them.
+
+Every event is required to have a related [place](#place) as its [location](#location), and can optionally have a related [organizer](#organizer).
+
+### Online event
+
+Same as an [event](#event), but happening online (as indicated by their `attendanceMode` property).
+
+For backward compatibility with older integrations that expect every event to have a [location](#location), online events are automatically related to the [nil location](#nil-location).
+
+## Place
+
+An API resource with a unique id that represents a place that can be visited and/or where [events](#event) can be located.
+
+If the place is published, it will also be published in [online calendars](#online-calendar) for interested visitors to find them, or to find other events located at the same place.
+
+Places created as a draft that do not get published can still be used as [locations](#location) for events, but will not be visible in search results in online calendars.
+
+## Offer
+
+The collective name both for events and places, usually used to indicate. Translated from the Dutch "Aanbod" to indicate the events/places that users of [online calendars](#online-calendar) can attend or visit.
+
+## Location
+
+A [place](#place) that a specific [event](#event) is happening at. The place will be included in the event's JSON as a `location` property.
+
+### Nil location
+
+A location for events that have no [physical location](#physical-location), like [online events](#online-event), to maintain compatibility with older integrations that expect every [event](#event) to have a location.
+
+*   Test: https://io-test.uitdatabank.be/place/00000000-0000-0000-0000-000000000000 
+*   Production: https://io.uitdatabank.be/place/00000000-0000-0000-0000-000000000000
+
+Online events will automatically become related to the nil location. Other events cannot be located at the nil location.
```

Comment:
The `Other events` can maybe be defined as "offline" or "mixed"?

Reply:
Good idea!